python3.8/cookiecutter-aws-sam-efs-python/{{cookiecutter.project_name}}/hello_efs/app.py

```diff
@@ -0,0 +1,28 @@
+import json
+from pathlib import Path
+
+# You can reference EFS files by including your local mount path, and then
+# treat them like any other file.
+FILE = Path("/mnt/lambda/file")
```

Comment:
We should add a comment here as this file might not be the present in the container.

Reply:
Added.